### PR TITLE
Implement policy_lb_client_ssl_profile resource

### PR DESCRIPTION
**Pull request by 2ez4szliu**
Implement nsxt_policy_lb_client_ssl_profile resource

### CODE REVIEW

File: nsxt/resource_nsxt_policy_lb_client_ssl_profile.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":       getNsxIDSchema(),
+			"path":         getPathSchema(),
+			"display_name": getDisplayNameSchema(),
+			"description":  getDescriptionSchema(),
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			"cipher_group_label": {
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_lb_client_ssl_profile.go:41]: Sounds from the description this should be Computed only?

Comment (annakhm) [reply nsxt/resource_nsxt_policy_lb_client_ssl_profile.go:41]: The doc says below `A label of cipher group which is mostly consumed by GUI. The server will populate this field when returing the resource`. So one of those seems incorrect - either the doc, or the code

Comment (2ez4szliu) [reply nsxt/resource_nsxt_policy_lb_client_ssl_profile.go:41]: I think the doc is incorrect, it's fixed now.

> The server will populate this field when returing the resource.  Ignored on PUT and POST.
These are comments for the Links field in the model package, not sure why it ended up in the cipher_group_label after auto-generating the doc. Thanks for catching this

File: nsxt/resource_nsxt_policy_lb_client_ssl_profile.go [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+func resourceNsxtPolicyLBClientSslProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyLBClientSslProfileCreate,
+		Read:   resourceNsxtPolicyLBClientSslProfileRead,
+		Update: resourceNsxtPolicyLBClientSslProfileUpdate,
+		Delete: resourceNsxtPolicyLBClientSslProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_lb_client_ssl_profile.go:31]: lets use `nsxtPolicyPathResourceImporter` for consistency

File: website/docs/r/policy_lb_client_ssl_profile.html.markdown [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_lb_client_ssl_profile"
+description: A resource to configure a LB Client SSL Profile.
+---
+
+# nsxt_policy_lb_client_ssl_profile
+
+This resource provides a method for the management of a LBClientSslProfile.
+
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
```
Comment (annakhm) [inline website/docs/r/policy_lb_client_ssl_profile.html.markdown:12]: Global manager is not supported

File: website/docs/r/policy_lb_client_ssl_profile.html.markdown [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+```hcl
+resource "nsxt_policy_lb_client_ssl_profile" "test" {
+  display_name          = "test"
+  description           = "Terraform provisioned LBClientSslProfile"
+  cipher_group_label    = "HIGH_COMPATIBILITY"
+  ciphers               = "TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA"
+  is_fips               = true
+  is_secure             = true
+  prefer_server_ciphers = true
+  protocols             = "TLS_V1_2"
```
Comment (annakhm) [inline website/docs/r/policy_lb_client_ssl_profile.html.markdown:25]: this is a list

File: website/docs/r/policy_lb_client_ssl_profile.html.markdown [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+This resource is applicable to NSX Global Manager, NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_lb_client_ssl_profile" "test" {
+  display_name          = "test"
+  description           = "Terraform provisioned LBClientSslProfile"
+  cipher_group_label    = "HIGH_COMPATIBILITY"
+  ciphers               = "TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA"
```
Comment (annakhm) [inline website/docs/r/policy_lb_client_ssl_profile.html.markdown:21]: this is a list

File: website/docs/r/policy_lb_client_ssl_profile.html.markdown (hunk 3 above)
Comment (2ez4szliu) [reply website/docs/r/policy_lb_client_ssl_profile.html.markdown]: Fixed

File: website/docs/r/policy_lb_client_ssl_profile.html.markdown (hunk 4 above)
Comment (2ez4szliu) [reply website/docs/r/policy_lb_client_ssl_profile.html.markdown]: Fixed

File: website/docs/r/policy_lb_client_ssl_profile.html.markdown (hunk 5 above)
Comment (2ez4szliu) [reply website/docs/r/policy_lb_client_ssl_profile.html.markdown]: Fixed

**Review by annakhm — APPROVED** [review]
Thank you!

**Pull request merged**